src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -951,10 +789,16 @@ public void TestConnectionStringEnvVarHasHighestPrecedence()
                 request.Headers.Add("user-agent", BROWSER_USER_AGENT_HEADER);
                 request.Headers.Add("accept", BROWSER_ACCEPT_HEADER);
 
+                // Adding the following headers simulates an interactive browser request.
+                request.Headers.Add("user-agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36");
+                request.Headers.Add("accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9");
+
```

Comment:
This looks like erroneous copy/paste of the two lines above this which use constants for header values.